### ISSUE TITLE
Discord Flag Fix

### ISSUE THREAD

**Issue by IntoxicatedSoftware745** [Rdimo/Hazard-Token-Grabber-V2#448]
Hello, I thought I'd just bring this to your attention, this is a sick script, but when I was reading through the code looking how it works, I noticed something that I, funnily enough, was working with today. 
The flags in the neatifyTokens function could be done differently as a user can have multiple flags/badges. An easy way to possibly redo this can be by checking the bin value of the flags. 
You see, I am not 100% sure how to explain this, but each slot is a unique flag and are sorted using bitwise ([the flags documented on discord](https://discord.com/developers/docs/resources/user#user-object-user-flags) and [all the flags](https://flags.lewistehminerz.dev)) therefore one way you can do this is check the different slots for a bool value of each (1 for True and 0 for False)
I know this is a super long thing for basically nothing, but I think that I might as well bring this your attention. Hope this helps in some way!

**Comment by GodFather227** [comment]
Fix it and make a pull damn

**Comment by IntoxicatedSoftware745** [comment]
oh, sorry, I'm pretty new to github, I'll do just that :)
Sorry for being newbie

**Comment by addi00000** [comment]
I actually made something like this for [my logger](https://github.com/addi00000/empyrean/blob/8c30c0152f5ed632f7f9af6ba5c8a9bba7004229/main.py#L51) if you would like to look.

Maybe you or I could make a pr.

**Comment by IntoxicatedSoftware745** [comment]
Already made a pull

**Comment by IntoxicatedSoftware745** [comment]
> I actually made something like this for [my logger](https://github.com/addi00000/empyrean/blob/8c30c0152f5ed632f7f9af6ba5c8a9bba7004229/main.py#L51) if you would like to look.
> 
> Maybe you or I could make a pr.

I mean, I'm down for that, I'm just really newbie when it comes to github

**Comment by Rdimo** [comment]
> Hello, I thought I'd just bring this to your attention, this is a sick script, but when I was reading through the code looking how it works, I noticed something that I, funnily enough, was working with today. The flags in the neatifyTokens function could be done differently as a user can have multiple flags/badges. An easy way to possibly redo this can be by checking the bin value of the flags. You see, I am not 100% sure how to explain this, but each slot is a unique flag and are sorted using bitwise ([the flags documented on discord](https://discord.com/developers/docs/resources/user#user-object-user-flags) and [all the flags](https://flags.lewistehminerz.dev)) therefore one way you can do this is check the different slots for a bool value of each (1 for True and 0 for False) I know this is a super long thing for basically nothing, but I think that I might as well bring this your attention. Hope this helps in some way!

yes I am aware of that. I tried to implement it ages ago and ended up pushing out tons of updates without fixing that flag thing. Ended up accidently deleting it and then I gave up
